contracts/phoenix-bonds/src/token_receiver.rs

```diff
@@ -0,0 +1,55 @@
+use crate::*;
+use near_contract_standards::fungible_token::receiver::FungibleTokenReceiver;
+use near_sdk::{near_bindgen, PromiseOrValue};
+
+const MINIMUM_BOND_LINEAR_AMOUNT: u128 = ONE_NEAR / 10; // 0.1 LiNEAR
```

Comment:
Should `MINIMUM_BOND_LINEAR_AMOUNT` be `0.11 LiNEAR`, ensuring that it exceeds minimum bond amount + storage cost?

Reply:
I still prefer 0.1 because 0.11 `LiNEAR` is even harder for users to understand, and given that LiNEAR price is above `1.09` now in mainnet, 0.1 `LiNEAR` is almost worth 0.11 `NEAR` to cover both minimum bond amount and storage cost.